docs/calibration-guide.md

```diff
@@ -0,0 +1,64 @@
+# Calibration guide
+This guide describes `recOrder`'s calibration routine with details about its goals, parameters, and evaluation metrics. 
+
+## Why calibrate?
+
+`recOrder` sends commands via MicroManager (or a TriggerScope) to apply voltages to the liquid crystals which modify the polarization of the light that illuminates the sample. `recOrder` could apply a fixed set of voltages so the user would never have to worry about these details, but this approach leads to extremely poor performance because
+
+- the sample, the sample holder, lenses, dichroics, and other optical elements introduce small changes in polarization, and 
+- the liquid crystals' voltage response drifts over time.
+
+Therefore, recalibrating the liquid crystals regularly (definitely between imaging sessions, often between different samples) is essential for acquiring optimal images. 
+
+## Finding the extinction state
+
+Every calibration starts with a routine that finds the **extinction state**: the polarization state (and corresponding voltages) that minimizes the intensity that reaches the camera. If the analyzer is a right-hand-circular polarizer, then the extinction state is the set of voltages that correspond to left-hand-circular light in the sample. 
+
+## Setting a goal for the remaining states: swing 
+
+After finding the circular extinction state, the calibration routine finds the remaining states. The **swing** parameter sets the target for the remaining states and is best understood using [the Poincare sphere](https://en.wikipedia.org/wiki/Unpolarized_light#Poincar%C3%A9_sphere), a diagram that organizes all pure polarization states onto the surface of a sphere.
```

Comment:
slight edit: The **swing** parameter sets the target ellipticity of the remaining states...